tests/unit/file_lib_test.c

```diff
@@ -1614,6 +1614,24 @@ static void test_file_read(void)
     return_to_test_dir();
 }
 
+static void test_read_file_stream_to_buffer(void)
+{
+    setup_tempfiles();
+
+    {
+        const size_t length = strlen(TEST_STRING);
+        char buf[1024] = {0};
+        FILE *const file = safe_fopen(TEST_FILE, "r");
+        assert_true(file != NULL);
+        const ssize_t read = ReadFileStreamToBuffer(file, length, buf);
```

Comment:
Please name the variable `n_read` or something that is not masking the `read()` function.

Reply:
Thanks, fixed.